x/wasm/client/cli/tx.go

```diff
@@ -21,13 +22,20 @@ import (
 )
 
 const (
-	flagTo     = "to"
-	flagAmount = "amount"
+	flagTo      = "to"
+	flagAmount  = "amount"
+	flagSource  = "source"
+	flagBuilder = "builder"
 )
 
 // limit max bytes read to prevent gzip bombs
 const maxSize = 400 * 1024
 
+// whitelist
+var validBuildTags = map[string]bool{
```

Comment:
Nice check, but I would prefer to whitelist all in valid format on the server, and let the clients evolve faster (rather than need a blockchain upgrade for 0.6.1 or 0.6.2